docs/styled.mdx

```diff
@@ -159,7 +159,7 @@ This API was inspired by [glamorous](https://github.com/paypal/glamorous). ❤
 
 ### Customizing prop forwarding
 
-By default, Emotion passes all props to custom components and only props that are valid html attributes for string tags. You can customise this by passing a custom `shouldForwardProp` function. You can also use `@emotion/is-prop-valid` (which is used by emotion internally) to filter out props that are not valid as html attributes.
+By default, Emotion passes all props except for `theme` to custom components and only props that are valid html attributes for string tags. You can customise this by passing a custom `shouldForwardProp` function. You can also use `@emotion/is-prop-valid` (which is used by emotion internally) to filter out props that are not valid as html attributes.
```

Comment:
I'm not sure if this is really that misleading. "all props" are mentioned as all given to the component, `theme` is usually injected to interpolations by emotion and not given explicitly by a user.

Reply:
Our use case is for a component library that exports a set of theme objects and components that expose a `theme` prop. These components use emotion internally, and it wasn't clear at first that we needed to set `{ shouldForwardProp: () => true }` so that the `theme` prop would be passed down into internal components.

Reply:
What do you think about such wording?
> By default, Emotion passes all externally received props to custom components and only props that are valid html attributes for string tags.

Reply:
In our case the `theme` prop is externally received. It doesn't seem to me that our case is that unusual.

Reply:
Hm, I believe that most people use the `theme` from the context and not from the external props, so it seems to me that your use case is a little bit unusual. I get what you are saying though, my version wouldn't be correct - because you actually pass `theme` externally.

Reply:
Couldn't it also be possible (though maybe unlikely) that someone could have a component that has a prop that happens to be named `theme`, but is unrelated to emotion's concept of a theme?

Reply:
It's possible - we are in JavaScript world after all 😅 This is completely not supported though and can break badly. It's also not allowed by provided TS types.